src/library/scala/collection/GenTraversableLike.scala

```diff
@@ -158,20 +158,6 @@ trait GenTraversableLike[+A, +Repr] extends Any with GenTraversableOnce[A] with
   @migration("The behavior of `scanRight` has changed. The previous behavior can be reproduced with scanRight.reverse.", "2.9.0")
   def scanRight[B, That](z: B)(op: (A, B) => B)(implicit bf: CanBuildFrom[Repr, B, That]): That
 
-  /** Applies a function `f` to all elements of this $coll.
```

Comment:
Question about applying this change to two other traits.

`TraversableLike` could also have this method and documentation deleted. It would inherit the method indirectly from `GenTraversableOnce`.

`TraversableOnce` also inherits `foreach` from `GenTraversableOnce` so it could be deleted as well.

Would this be safe to do?